apps/hash-api/src/integrations/linear/oauth.ts

```diff
@@ -237,29 +253,58 @@ export const oAuthLinearCallback: RequestHandler<
         entityTypeIds: [systemEntityTypes.linearIntegration.entityTypeId],
         webId: userAccountId as WebId,
         properties: linearIntegrationProperties,
-        relationships: createDefaultAuthorizationRelationships(authentication),
+        relationships: [
+          ...createDefaultAuthorizationRelationships(authentication),
+          {
+            relation: "editor",
+            subject: {
+              kind: "account",
+              subjectId: linearBotAccountId,
+            },
+          },
+        ],
       },
     );
 
+    const entityUuid = extractEntityUuidFromEntityId(
+      linearIntegrationEntity.entityId,
+    );
+
+    await createPolicy(req.context.graphApi, authentication, {
+      name: `linear-integration-bot-view-entity-${entityUuid}`,
+      principal: {
+        type: "actor",
+        actorType: "machine",
+        id: linearBotAccountId,
+      },
+      effect: "permit",
+      actions: ["viewEntity"],
+      resource: {
+        type: "entity",
+        id: entityUuid,
+      },
+    });
+
+    await createPolicy(req.context.graphApi, authentication, {
```

Comment:
see https://github.com/hashintel/hash/pull/7405
```suggestion
    // TODO: allow creating policies alongside entity creation
    //   see https://linear.app/hash/issue/H-4622/allow-creating-policies-alongside-entity-creation
    await createPolicy(req.context.graphApi, authentication, {
```